autoload/clap/animate.vim

```diff
@@ -0,0 +1,76 @@
+
+let g:clap_animation_frequency = get(g:, 'clap_animation_frequency', 50)
+
+function! clap#animate#start(duration, initial, final, type, Fn)
+  let duration = a:duration
+  let initial  = a:initial
+  let final    = a:final
+  let type     = a:type
```

Comment:
This does not look very helpful, just using `a:*` is good.

Reply:
Yes I was playing with the parameters format (between a dict or like that), I'll remove those.